specification/oracle/Oracle.Database/operations/autonomousdatabase/autonomousDatabaseBackupOperations.tsp

```diff
@@ -0,0 +1,43 @@
+import "../../../models/autonomousdatabase/autonomousDatabaseBackup.tsp";
+import "../commons.tsp";
+
+using Azure.ResourceManager;
+using Autorest;
+using TypeSpec.Versioning;
+
+namespace Oracle.Database;
+
+@armResourceOperations(AutonomousDatabaseBackup)
+interface AutonomousDatabaseBackups
+  extends OracleChildResource<
+      AutonomousDatabaseBackup,
+      AutonomousDatabaseBackupProperties
+    > {
+  @example(
```

Comment:
since all you example files have `operationId` and `title`, you don't need to explicitly set `@example`. instead, you should update `title` part in example.

Reply:
Thank you, can we take it as an advice for next PR to improve?